pages/apim/installation-guide/installation-guide-migration.adoc

```diff
@@ -0,0 +1,6 @@
+:page-sidebar: apim_sidebar
+:page-permalink: apim_installguide_migration.html
+:page-folder: apim/installation-guide
+:page-toc: true
+
+include::https://raw.githubusercontent.com/gravitee-io/release/master/upgrades/README.adoc[]
```

Comment:
Please add a sentence to explain that files like `upgrades/1.4.0/mongodb/1-update-application.js` can be found at https://github.com/gravitee-io/release/tree/master/upgrades